content/features/featuresDeepDive/mesh/gaussianSplatting.md

```diff
@@ -28,6 +28,50 @@ BABYLON.SceneLoader.ImportMeshAsync(null, "https://assets.babylonjs.com/splats/"
     const gaussianSplattingMesh = result.meshes[0]; });
 ```
 
+## Updating datas of a Gaussian Splatting
+
+User can update or generate new Splats with the `updateData`method.
+Mandatory values are splat center position, size, orientation and color.
+The following example updates the Gaussian Splatting with 1 single 

```

Comment:
I think the end of the sentence is missing?

Reply:
Indeed! Fixed